### ISSUE TITLE
Handing duplicated initial authors in retrieve_pubmed_records

### ISSUE THREAD

**Issue by bryan-brancotte** [kjolley/BIGSdb#767]
Hello Keith

After the refactoring made for https://github.com/kjolley/BIGSdb/issues/611 I am starting to use it (very lately I agree).

I have an error : 
```
ERROR - DBD::Pg::db do failed: ERROR:  duplicate key value violates unique constraint "refauthors_pkey"
DETAIL:  Key (pmid, author)=(24407631, 1808) already exists. at /usr/local/lib/BIGSdb/Datastore.pm line 2120.
```

It is caused by https://pubmed.ncbi.nlm.nih.gov/24407631/ where two `Zhang X` can be found in the authors.

There is no rush in fixing it as the rest is updated, but still it shows an error so here I am.

Best regards

**Comment by kjolley** [comment]
This was fixed in v1.7.1, where the primary key for the refsauthors table was modified to include pmid, author, and position fields. https://github.com/kjolley/BIGSdb/commit/4e6426913a9945b383180fa2f70a5c3006d324a1. There doesn't seem to have been a SQL file to make the update to the refs database though (sorry about that).

You can manually update the refs database with the following:

```
ALTER TABLE refauthors DROP CONSTRAINT refauthors_pkey;
ALTER TABLE refauthors ADD PRIMARY KEY (pmid,author,position);
```

**Comment by bryan-brancotte** [comment]
Thanks ! (we've just applied it)
